packages/lib-react-components/src/PlainButton/PlainButton.js

```diff
@@ -28,7 +28,7 @@ function PlainButton (props) {
     color,
     ...rest
   } = props
-  const renderAs = as || href && disabled && 'span'
+  const renderAs = (href && disabled) ? 'span' : as
```

Comment:
I had to change how this conditional works because previously the falsey condition returned false which is an invalid prop for `as`. Now the default prop of an empty string is used if as isn't defined or if href and disabled isn't defined.